Sources/Fullscreen/EmptyView/EmptyView.swift

```diff
@@ -20,6 +21,9 @@ public class EmptyView: UIView {
     private let sizeOfSquare = CGSize(width: 50, height: 50)
 
     private var hasLayedOut = false
+    private let isChristmasThemed: Bool
```

Comment:
I would probably make a separate view or something like this, but this flag should work for now

Reply:
Yes, I thought of making a separate view, but that means more work in the iOS app just to remove it when christmas is over.